app/config/sui/contracts-mainnet.json

```diff
@@ -12,5 +12,8 @@
 		"bidEndpoint": "bid",
 		"withdrawFunction": "withdraw",
 		"auctionId": "0x161524be15687cca96dec58146568622458905c30479452351f231cac5d64c41"
+	},
+	"nBTC": {
+		"suiScanExplorerLink": "https://suiscan.xyz/mainnet/account/"
 	}
```

Comment:
```suggestion
	"accountExplorer": "https://suiscan.xyz/mainnet/account/"
```

Reply:
This is not a specific to nBTC, but for any account.

Reply:
Update the testnet link as well.

Reply:
Done